.github/actions/build_node/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM node:10
```

Comment:
A `node:10-slim` image will be a lot smaller, so faster to run in Actions. The main difference I know of is that `-slim` doesn't have `git`, so dependencies not from NPM won't install